requirements/test-requirements.in

```
@@ -1,3 +1,4 @@
-pytest==7.1.1
-pytest-testinfra==6.0.0
+docker==6.0.0
+pytest==7.1.3
+pytest-testinfra==6.8.0
 PyYAML==6.0
```

`requirements.in` should not be pinned.
```suggestion
docker
pytest
pytest-testinfra
PyYAML
```